src/Akeneo/Bundle/StorageUtilsBundle/Elasticsearch/Cursor/Cursor.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace Akeneo\Bundle\StorageUtilsBundle\Elasticsearch\Cursor;
+
+use Akeneo\Component\StorageUtils\Cursor\CursorInterface;
+use Akeneo\Component\StorageUtils\ElasticsearchClientInterface;
+use Akeneo\Component\StorageUtils\Repository\CursorableRepositoryInterface;
+
+/**
+ * Cursor to iterate on items
+ *
+ * @author    Marie Bochu <marie.bochu@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class Cursor implements CursorInterface
```

Comment:
maybe we can remove the ES client interface 
and put the cursor in the StorageUtilsBundle